osu.Game/Online/API/Requests/Responses/SoloScoreInfo.cs

```diff
@@ -27,9 +27,6 @@ public class SoloScoreInfo : IScoreInfo
         [JsonProperty("build_id")]
         public int? BuildID { get; set; }
 
-        [JsonProperty("passed")]
-        public bool Passed { get; set; }
```

Comment:
Well for one thing I can certainly tell you that this cannot fly.

https://github.com/ppy/osu-web/blob/f73ea11049f333731c09ec9b1b6c1141a65a8320/app/Models/Solo/Score.php#L85-L90
https://github.com/ppy/osu-web/blob/f73ea11049f333731c09ec9b1b6c1141a65a8320/app/Models/Solo/Score.php#L123

Removing this flag will likely break things to the point wherein no submitted score will have `preserved` set. If this diff is to proceed we will need to wean off web from the flag first and deploy that change.

Reply:
I want to say I miswrote the PR's description, I meant to say all consumers of `SoloScoreInfo` should be *changed* to no longer use the property...totally my bad though.

I saw those usages, and that's actually exactly why I didn't pursue this any further and left this as a task for someone more knowledgable in osu-web to change things around there (replace `passed` with `rank != F`).